src/main/scala/nl.knaw.dans.easy.dd2d/dataverse/json/package.scala

```diff
@@ -15,21 +15,19 @@
  */
 package nl.knaw.dans.easy.dd2d.dataverse
 
-import better.files.File
+import nl.knaw.dans.lib.dataverse.model.dataset.{ CompoundField, ControlledMultipleValueField, ControlledSingleValueField, MetadataField, PrimitiveMultipleValueField, PrimitiveSingleValueField }
 import org.json4s.{ DefaultFormats, Formats }
 
 import scala.collection.mutable
 
 package object json {
```

Comment:
Maybe we coudl get rid of this package altogether. The FieldMap is still handy though. In any case, we should move it. There is now an empty parent package "dataverse".  FieldMap is exclusively used from the mapping package, so how about moving it to the package object of that package?